### PR TITLE
📝 adding regular sprint maintenance template

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- 📝 Adding issue template for maintenance that should be done on regular basis (currently includes upgrade & unifying of most important libraries)
- 📝 Minor changes in other templates

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/4_pre_release.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: 🚀 Pre-release to staging (developers-only)
 description: Issue to plan and log pre-release from master to staging deploy (including staging hotfixes)
 title: "🚀 Pre-release master -> staging_<sprint_name><version>"
-labels: ["t:maintenance", "release"]
+labels: ["t:maintenance", "Release"]
```
Comment (sanderegg) [inline .github/ISSUE_TEMPLATE/4_pre_release.yml:4]: Q: is it good that ```Release``` becomes the only label with a capital letter? Just wondering.

File: .github/ISSUE_TEMPLATE/5_release.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ body:
       description: |
         Changes introduced by this release.
         List links to the PRs associated to every commit in the release. This will automatically create references between PRs and this release that will be very valuable for traceability.
-        TIP: use the autogenerate changelog feature during the draft release.
+        TIP: use the autogenerate changelog feature during the draft release. (TIP: use regex replace for better visualization ``\*.+ by (@\w+) in (https:.*)`` -> ``* $2 by $1``)
```
Comment (sanderegg) [inline .github/ISSUE_TEMPLATE/5_release.yml:61]: use regex replace. how so? is this in bash?

**Review by sanderegg — APPROVED** [review]
thanks. maybe just consider my comment about having a capital letter or not.

File: .github/ISSUE_TEMPLATE/5_release.yml (hunk 2 above)
Comment (matusdrobuliak66) [reply .github/ISSUE_TEMPLATE/5_release.yml:61]: No that is just a tip that i wanted to document somewhere for myself, when you are creating a release issue, and you copy&paste there the change log from the draft tag, it doesn't looks nice as the information there is duplicated. This is just a helper how to make it nicer.

File: .github/ISSUE_TEMPLATE/4_pre_release.yml (hunk 1 above)
Comment (matusdrobuliak66) [reply .github/ISSUE_TEMPLATE/4_pre_release.yml:4]: Ok i had changed the label definition to lower case to be consistent

**Pull request merged**